MovieEnthusiast.Infrastructure/Mapping/MovieProfile.cs

```diff
@@ -0,0 +1,16 @@
+﻿using AutoMapper;
```

Comment:
[CleanArchitecture](https://github.com/jasontaylordev/CleanArchitecture) template has Automapper Profiles inside the Application layer. I think it also makes sense as you already have Automapper dependency in that layer. I suggest moving mapping profiles to the Application layer.